schemas/mutations.graphql

```diff
@@ -0,0 +1,3 @@
+type Mutation {
+    

```

Comment:
Implement your first mutation example: 
- `createQuestion(form: CreateQuestion): Question` with `Input CreateQuestion` in `input.graphql` file